sentry/src/routes.rs

```diff
@@ -81,9 +96,15 @@
 //!
 //! #### GET `/v5/channel/:id/spender/all` (auth required)
 //!
+//! This routes gets total_deposited and total_spent for every spender on a [`Channel`]
+//!
 //! The route is handled by [`channel::get_all_spender_limits()`].
 //!
-//! Response: [`AllSpendersResponse`](primitives::sentry::AllSpendersResponse)
+//! Response:
```

Comment:
Please keep the "Response: [`AllSpendersResponse`](primitives::sentry::AllSpendersResponse)" and include an `Examples` section